src/lava/lib/dnf/connect/connect.py

```diff
@@ -39,6 +41,10 @@ def connect(
     ops : list(AbstractOperation), optional
         list of operations that describes how the connection between
         <src_op> and <dst_ip> will be created
+    connection : Connection process of type Dense or Sparse, optional
+        Process used between src_op and dst_ip. If connection is None
+        the connection process will be defined automatically (currently a Sparse
+        Process is always used).
```

Comment:
This is inaccurate. You are not expecting a Process (instance) here but a class, either `Dense` or `Sparse`.